nx.json

```diff
@@ -0,0 +1,119 @@
+{
+  "targetDefaults": {
```

Comment:
These targetDefaults are the targets one can target with `nx run-many` or `nx affected`